var/spack/repos/builtin/packages/py-torch/package.py

```diff
@@ -104,7 +104,12 @@ class PyTorch(PythonPackage, CudaPackage, ROCmPackage):
     )
 
     conflicts("+cuda+rocm")
-    conflicts("+tensorpipe", when="+rocm", msg="TensorPipe doesn't yet support ROCm")
+    conflicts("+tensorpipe", when="+rocm@:5.1")
```

Comment:
This constraint doesn't seem correct...what are you trying to say here?

Reply:
Note that `+rocm@:5.1` is equivalent to `@:5.1 +rocm`, so this indeed doesn't make sense